NETCORE/NETCORE/Controllers/MembersController.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using NETCORE.DatabaseAccess.Models;
+using NETCORE.DatabaseAccess.Repositories;
+
+namespace NETCORE.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class MembersController : ControllerBase
+    {
+        private readonly IMemberRepository memberRepo;
+        public MembersController(IMemberRepository repository)
+        {
+            memberRepo = repository;
+        }
+
+        // GET: api/Members
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<Member>>> GetMembers()
+        {
+            return await memberRepo.GetAll();
```

Comment:
Have to implement a MemberService to interact with MemberRepsitory:
Controller => Service => Repository